msg/msg.go

```diff
@@ -39,6 +75,110 @@ func (c CTCP) Value() string {
 }
 
 // Decode turns the input message from a CTCP escaped message into plain text.
-func Decode(string) ([]Valuer, error) {
-	return nil, errors.New("not implemented")
+func Decode(message string) ([]Valuer, error) {
```

Comment:
If it accepts a CTCP-escaped message, why CTCP as the message's type? Or change this comment to be more accurate.

Reply:
It has been updated